VRDR.HTTP/Nightingale.cs

```diff
@@ -463,36 +473,35 @@ public static DeathRecord FromNightingale(string json)
                     mar.Add("display", "Married");
                     deathRecord.MaritalStatus = mar;
                     break;
-                case "Married but seperated":
+                case "Legally Separated":
                     mar = new Dictionary<string, string>();
                     mar.Add("code", "L");
                     mar.Add("system", "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus");
                     mar.Add("display", "Legally Separated");
                     deathRecord.MaritalStatus = mar;
                     break;
                 case "Widowed":
-                case "Widowed (but not remarried)":
                     mar = new Dictionary<string, string>();
                     mar.Add("code", "W");
                     mar.Add("system", "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus");
                     mar.Add("display", "Widowed");
                     deathRecord.MaritalStatus = mar;
                     break;
-                case "Divorced (but not remarried)":
+                case "Divorced":
                     mar = new Dictionary<string, string>();
                     mar.Add("code", "D");
                     mar.Add("system", "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus");
                     mar.Add("display", "Divorced");
                     deathRecord.MaritalStatus = mar;
                     break;
-                case "Never married":
+                case "Never Married":
```

Comment:
Good catch on these, but maybe it would have been better to have originally made the switch param `.ToLower`, and have these lower case.